gateway/policy.go

```diff
@@ -31,6 +31,9 @@ type DBAccessDefinition struct {
 	DisableIntrospection bool                         `json:"disable_introspection"`
 	FieldAccessRights    []user.FieldAccessDefinition `json:"field_access_rights"`
 	Limit                *user.APILimit               `json:"limit"`
+
+	// Endpoints contains endpoint rate limit settings.
+	Endpoints []user.Endpoint `json:"endpoints,omitempty" msg:"endpoints,omitempty"`
```

Comment:
what's the msg tag, why do other fields not have it?

Reply:
typo, copied over from session.go, this is used for dashboard interactions